src/app/pages/storage/volumes/volumes-list/components/export-disconnect-modal.component.ts

```diff
@@ -0,0 +1,313 @@
+    this.form.get('nameInput').setErrors(null);
+  }
+
+  private getPoolSummary(
+    pool: VolumesListPool,
+    attachments: PoolAttachment[],
+    processes: PoolProcess[],
+  ): string {
+    let summary = '';
+
+    if (attachments.length) {
+      summary += this.translate.instant(helptext.exportMessages.services, { name: pool.name });
+      summary += this.getSummaryAttachments(attachments);
+    }
+
+    if (processes.length) {
+      summary += this.getSummaryProcesses(processes, pool);
+    }
+
+    return summary;
+  }
+
+  private getSummaryAttachments(attachments: PoolAttachment[]): string {
+    function formatAttachmentLine(line: string): string {
+      return `<br> - ${line}`;
+    }
+    function formatAttachmentPart(part: string): string {
+      return part.split(',').map((line) => formatAttachmentLine(line)).join('');
+    }
+    function formatAttachment(a: PoolAttachment): string {
+      return `<br><b>${a.type}:</b>` + a.attachments.map((x) => formatAttachmentPart(x)).join('');
+    }
+
+    const summary = attachments.map((a) => formatAttachment(a)).join('');
+    return summary + '<br /><br />';
+  }
+
+  private getSummaryProcesses(processes: PoolProcess[], pool: VolumesListPool): string {
+    const running: PoolProcess[] = [];
+    const unknown: PoolProcess[] = [];
+    processes.forEach((p) => {
+      if (!p.service && p.name) {
+        running.push(p);
+      } else {
+        unknown.push(p);
+      }
+    });
+
+    function formatProcessLine(p: PoolProcess): string {
+      if (!p.service && p.name) {
+        return `<br> - ${p.name}`;
+      }
+      return `<br> - ${p.pid} - ${p.cmdline.substring(0, 40)}`;
+    }
+
+    let summary = '';
+
+    if (running.length) {
+      let runningSummary = this.translate.instant(helptext.exportMessages.running);
+      runningSummary += `<b>${pool.name}</b>:`;
+      runningSummary += running.map((x) => formatProcessLine(x)).join('');
+
+      summary += runningSummary;
+    }
+
+    if (unknown.length) {
+      let unknownSummary = '<br><br>' + this.translate.instant(helptext.exportMessages.unknown);
+      unknownSummary += unknown.filter((p) => p.pid).map((p) => formatProcessLine(p)).join('');
+      unknownSummary += '<br><br>' + this.translate.instant(helptext.exportMessages.terminated);
+
+      summary += unknownSummary;
+    }
+
+    return summary;
+  }
+
+  private startExportDisconnectJob(pool: VolumesListPool, value: ExportDisconnectFormValue): void {
+    const entityJobRef = this.matDialog.open(EntityJobComponent, {
+      data: { title: helptext.exporting },
+      disableClose: true,
+    });
+    entityJobRef.updateSize('300px');
+    entityJobRef.componentInstance.setDescription(helptext.exporting);
+    entityJobRef.componentInstance.setCall('pool.export', [pool.id, {
+      destroy: value.destroy,
+      cascade: value.cascade,
+      restart_services: this.restartServices,
+    }]);
+    entityJobRef.componentInstance.submit();
```

Comment:
`submit` should be called after success and failure has been defined. Because it leaves a potential for an event to miss the pipes. Even though the chance is very small.